lib/common-types.ts

```diff
@@ -566,6 +580,27 @@ export type MRActions = {
       return: AssetKeyProps
     }
   }
+  BulkAction: {
+    get: {
+      params: GetBulkActionParams
+      return: BulkActionProps
```

Comment:
`BulkActionProps` needs to be generic so that the payload shape can be specified